core/types/src/markers.rs

```diff
@@ -17,3 +17,15 @@ pub struct View;
 /// Spend key marker type
 #[derive(Copy, Clone, PartialEq, Debug)]
 pub struct Spend;
+
+/// Transaction key marker type
+#[derive(Copy, Clone, PartialEq, Debug)]
```

Comment:
Optional/Nit: Sort derives, throughout this file for consistency
```suggestion
#[derive(Clone, Copy, Debug, PartialEq)]
```

Reply:
Is that something we can add to our linter specs?

Reply:
TIL the order of derives is actually semantically important, [each derive can only "see" the derived in front of it](https://github.com/rust-lang/rustfmt/issues/1867), so e.g. in the original code here `#derive(Copy, Clone, PartialEq, Debug)` then `PartialEq` can only see that the type is also `Debug`.

Reply:
(Therefore, no, we sadly can't add this to our rustfmt, although maybe there's an annoying clippy warning we could enable.)